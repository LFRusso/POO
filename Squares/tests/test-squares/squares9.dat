Square0 -1.59849 -2.05691 0.712616
Square1 2.9411 2.76155 1.65998
Square2 1.73425 2.17929 3.49034
Square3 -2.89744 -1.23198 3.2413
Square4 0.962171 -0.732573 4.20276
Square5 1.41227 2.11074 1.41652
Square6 -1.20647 0.00934794 0.946481
Square7 0.464454 0.352862 2.04438
Square8 -1.7911 -1.54869 4.67003
Square9 -0.454183 1.6317 4.92325
