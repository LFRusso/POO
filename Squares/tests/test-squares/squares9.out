Square0 intercepts Square8 with area 0.145656
Square1 intercepts Square2 with area 2.75553
Square1 intercepts Square4 with area 1.17632
Square1 intercepts Square9 with area 2.53639
Square2 intercepts Square4 with area 4.42866
Square2 intercepts Square5 with area 1.47541
Square2 intercepts Square7 with area 0.168822
Square2 intercepts Square8 with area 1.07835
Square2 intercepts Square9 with area 9.54544
Square3 intercepts Square6 with area 0.895826
Square3 intercepts Square8 with area 6.92005
Square3 intercepts Square9 with area 0.301357
Square4 intercepts Square5 with area 1.92568
Square4 intercepts Square7 with area 3.16197
Square4 intercepts Square8 with area 7.38702
Square4 intercepts Square9 with area 6.44738
Square5 intercepts Square7 with area 0.314168
Square5 intercepts Square8 with area 1.43154
Square5 intercepts Square9 with area 2.00653
Square6 intercepts Square8 with area 0.895826
Square7 intercepts Square8 with area 4.17949
Square7 intercepts Square9 with area 1.56506
Square8 intercepts Square9 with area 4.96514
